Externals/wxWidgets3/include/wx/msw/app.h

```diff
@@ -29,7 +29,7 @@ class WXDLLIMPEXP_CORE wxApp : public wxAppBase
     virtual ~wxApp();
 
     // override base class (pure) virtuals
-    virtual bool Initialize(int& argc, wxChar **argv);
+    virtual bool Initialize(int& _argc, wxChar **_argv);
```

Comment:
Starting with VS2015, compiling wxwidgets causes many more warnings to be issued. I think wxwidgets was the only library in Externals which was not using `ClDisableAllWarnings.props`, but now I've added it.  
The patches to wxw headers are there because they cause warnings when dolphin libs include these headers (as opposed to when wxw is building). I don't see how we can avoid patching that, unless we `#pragma warning(push)` around inclusion of wxw headers, which is probably not a good idea.  
This specific warning was that arg is also a class member name.